deltachat-ios/Controller/AccountSetup/Instand Onboarding/InstantOnboardingViewController.swift

```diff
@@ -90,13 +132,35 @@ class InstantOnboardingViewController: UIViewController, ProgressAlertHandler {
     }
 
     @objc private func showPrivacy(_ sender: UIButton) {
-        guard let url = URL(string: "https://nine.testrun.org/privacy.html") else { return }
+        let url = providerHostURL.appendingPathComponent("/privacy.html")
 
         if UIApplication.shared.canOpenURL(url) {
             UIApplication.shared.open(url)
         }
     }
 
+    @objc private func showOtherOptions(_ sender: UIButton) {
+
+        guard let url = URL(string: "https://delta.chat/en/chatmail") else { return }
+
+        if UIApplication.shared.canOpenURL(url) {
+            UIApplication.shared.open(url)
+        }
+    }
+
+    @objc private func scanQRCode(_ sender: UIButton) {
+        let qrReader = QrCodeReaderController(title: String.localized("multidevice_receiver_title"),
+                    addHints: "➊ " + String.localized("multidevice_same_network_hint") + "\n\n"
+                        +     "➋ " + String.localized("multidevice_open_settings_on_other_device") + "\n\n"
+                        +     String.localized("multidevice_experimental_hint"),
+                    showTroubleshooting: true)
```

Comment:
title/hints should be like the following:

```suggestion
        let qrReader = QrCodeReaderController(title: String.localized("scan_invitation_code"))
```

(taking from removed https://github.com/deltachat/deltachat-ios/pull/2163/files#diff-d82264e2f2825dbdcaa4e863471e22a9b8e3bb46ee19e7b22528427ab6adcc86L38 )